lms/djangoapps/grades/models.py

```diff
@@ -257,8 +257,14 @@ def save_grade(cls, **kwargs):
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info("persistent subsection grade successfully saved: {0}".format(unicode(grade)))
```

Comment:
This line is responsible for the increased query count in the tests below. When we call `unicode` on the `grade` object, it makes a DB call to get its visible blocks.